### PR TITLE
pin scaffold deps

### PR DESCRIPTION
**Pull request by marcofranssen**
- Allow committing Chart.lock, exclude charts/*.tgz
- Pin scaffold dependency versions
- Add fulcio README.md.gotmpl and generate docs

 **Description of the change**

 **Existing or Associated Issue(s)**

#298 

 **Additional Information**

 

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [x] Variables are documented in the `values.yaml` and added to the README.md. THe [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content
- [x] JSON Schema generated
- [x] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command

### CODE REVIEW

File: ct.yaml [hunk 1]
```diff
@@ -3,3 +3,6 @@ chart-dirs:
 validate-maintainers: false
 remote: origin
 target-branch: main
+chart-repos:
+  - sigstore=https://sigstore.github.io/helm-charts
+
```
Comment (cpanato) [inline ct.yaml:8]: nit: extra line here

File: charts/scaffold/Chart.lock [hunk 2]
```diff
@@ -0,0 +1,15 @@
+dependencies:
```
Comment (cpanato) [inline charts/scaffold/Chart.lock:1]: same question @sabre1041 , is that ok

**Review by cpanato — CHANGES_REQUESTED** [review]
small nit

File: ct.yaml (hunk 1 above)
Comment (marcofranssen) [reply ct.yaml:8]: It is a good practice to end files with a whiteline. especially useful when `cat` in the terminal.

File: charts/scaffold/Chart.lock (hunk 2 above)
Comment (marcofranssen) [reply charts/scaffold/Chart.lock:1]: Probably good to mention is that it is easy to manage this see https://github.com/sigstore/helm-charts/pull/300 on how folks can update when bumping the dependency versions.

**Comment by marcofranssen** [comment]
Handled the feedback. Thanks for catching.

**Comment by hectorj2f** [comment]
@marcofranssen We have bumped the deps in scaffold.

**Review by priyawadhwa — APPROVED** [review]

**Review by priyawadhwa — APPROVED** [review]

**Review by cpanato — APPROVED** [review]

**Comment by cpanato** [comment]
@marcofranssen can we have a rebase just to make sure everything is up to date? thanks

**Comment by marcofranssen** [comment]
@cpanato done the rebase and regenerated the lock file based on new contents that came in from main.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Comment by sabre1041** [comment]
@marcofranssen it looks good. can you resolve this one final conflict and then we can merge it ASAP so you don't have to address this anymore

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**